__tests__/app.test.js

```diff
@@ -16,6 +19,99 @@ afterAll(() => {
 
 describe("news-api", () => {
     describe("GET requests", () => {
+        describe("GET /api/articles", () => {
+            test("responds with status code 200", () => {
+                return request(app)
+                .get("/api/articles")
+                .expect(200)
+            })
+            test("responds with an object with a key of 'articles'", () => {
+                return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body}) => {
+                    expect(body).toHaveProperty("articles")
+                })
+            })
+            test("the value of 'articles' is an array of objects", () => {
+                return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body}) => {
+                    const { articles } = body
+                    expect(articles.length).toBeGreaterThan(0)
+                    expect(articles instanceof Array).toBe(true)
+                    articles.forEach((element) => {
+                        expect(element instanceof Object).toBe(true)
+                    })
+                })
+            })
```

Comment:
Many of these tests can be combined and dont need to be broken up in this way, this would save the test from having to make a lot of requests just to check simple things.